assets/post_install_macOS.sh

```diff
@@ -28,23 +28,23 @@ else
     APP_DIR="$HOME"/Applications
     PERMS=""
 fi
-MNE_APP_DIR_ROOT="${APP_DIR}/MNE-Python"
-MNE_APP_DIR="${MNE_APP_DIR_ROOT}/${MNE_VERSION}"
-logger -p 'install.info' "📓 MNE_APP_DIR=$MNE_APP_DIR"
+SPI_APP_DIR_ROOT="${APP_DIR}/Scientific-Python"
+SPI_APP_DIR="${SPI_APP_DIR_ROOT}/${SPI_VERSION}"
+logger -p 'install.info' "📓 SPI_APP_DIR=$SPI_APP_DIR"
 
-logger -p 'install.info' "ℹ️ Moving root MNE .app bundles from $APP_DIR to $MNE_APP_DIR."
-$PERMS mv "$APP_DIR"/*\(MNE\).app "$MNE_APP_DIR"/
+logger -p 'install.info' "ℹ️ Moving root Scientific Python .app bundles from $APP_DIR to $SPI_APP_DIR."
+$PERMS mv "$APP_DIR"/*\(ScientificPython\).app "$SPI_APP_DIR"/
 
-logger -p 'install.info' "ℹ️ Fixing permissions of MNE .app bundles in $MNE_APP_DIR: new owner will be ${USER_FROM_HOMEDIR}."
-$PERMS chown -R "$USER_FROM_HOMEDIR" "$MNE_APP_DIR"
+logger -p 'install.info' "ℹ️ Fixing permissions of Scientific Python .app bundles in $SPI_APP_DIR: new owner will be ${USER_FROM_HOMEDIR}."
+$PERMS chown -R "$USER_FROM_HOMEDIR" "$SPI_APP_DIR"
 
-MNE_ICON_PATH="$PREFIX/Menu/mne.png"
-logger -p 'install.info' "ℹ️ Setting custom folder icon for $MNE_APP_DIR and $MNE_APP_DIR_ROOT to $MNE_ICON_PATH."
-for destPath in "$MNE_APP_DIR" "$MNE_APP_DIR_ROOT"; do
-    logger -p 'install.info' "ℹ️ Setting custom folder icon for $destPath to $MNE_ICON_PATH."
+SPI_ICON_PATH="$PREFIX/Menu/mne.png"  # TODO FIXME where is this image coming from / what would be the equivalent for Scientific Python?
```

Comment:
I *think* it would be this:

https://github.com/conda-forge/mne-feedstock/blob/main/recipe/menu/mne.png

That's an asset that's part of the [`mne-installer-menus`](https://anaconda.org/conda-forge/mne-installer-menus/files) feedstock:

https://github.com/conda-forge/mne-feedstock/blob/08355b8a5cd76fec2ae7d4e3d16245fdc754da5a/recipe/meta.yaml#L122

So this post-install script knows that the `mne-installer-menus` will be installed, and that its `mne.png` will land in `Menu/mne.png`. So we'll need to see if there is some place we can get a scientific Python PNG... if not, then we could create a conda-forge feedstock for it. Not sure what conda-forge project / recipe we would want to create for it...